src/Bucket.ts

```diff
@@ -111,8 +112,20 @@ export class Bucket {
             Bucket: this.name,
             Key: objectKey,
         });
-        const signedUrl = await getSignedUrl(this.r2, obj, { expiresIn });
-        return signedUrl;
+        return getSignedUrl(this.r2, obj, { expiresIn });
+    }
+
+    /**
+     * Returns the signed URL of an object used to perform a "PUT" action.
+     * @param objectKey
+     * @param expiresIn Expiration time in seconds.
+     */
+    public async putObjectSignedUrl(objectKey: string, expiresIn: number): Promise<string> {
```

Comment:
Isn't this just the same as the GET one? Or is it used when you want to put an object and then retrieve the signed URL?

Reply:
from the docs it seems like the GET and POST ones are different URLs. So it's used to generate a signed URL where you can PUT an object.